controlpanel/api/models/tool.py

```diff
@@ -204,3 +204,48 @@ def restart(self, id_token):
         """
 
         cluster.ToolDeployment(self.user, self.tool).restart(id_token)
+
+
+class HomeDirectory:
+    """
+    Represents a user's home directory in the cluster
+    """
+
+    def __init__(self, user):
+        self._subprocess = None
+        self.user = user
+
+    def __repr__(self):
+        return f"<HomeDirectoryManager: {self.user!r}>"
```

Comment:
Is the trailing `Manager` intentional? I guess it doesn't make much difference but it may confuse one of us in the future is the class name is slightly off 🤷‍♂️